Microsoft.Toolkit.Uwp/Extensions/StringExtensions.cs

```diff
@@ -18,27 +11,21 @@ namespace Microsoft.Toolkit.Uwp.Extensions
     /// </summary>
     public static class StringExtensions
     {
-        private static ResourceLoader _independentLoader = ResourceLoader.GetForViewIndependentUse();
-
```

Comment:
This was an optimization for calling GetLocalized more than once so we don't go refetch it and create new instances with each call.